config/base/params.env

```diff
@@ -1,12 +1,12 @@
-IMAGES_APISERVER=quay.io/opendatahub/ds-pipelines-api-server@sha256:4650c62254cd79112de3e4f09270130501d0d86a4dea79b74c2fcb8b5ca567e7
-IMAGES_ARTIFACT=quay.io/opendatahub/ds-pipelines-artifact-manager@sha256:58a13845901f8aae5421f640eeebee0abf3b12b27c1f96fbc8ff199b7e4f8d8d
-IMAGES_PERSISTENTAGENT=quay.io/opendatahub/ds-pipelines-persistenceagent@sha256:c8b0953c28fd24180ddd24a30c68df411d299ccc7f6bc18ab15f4dba4a84b7d9
-IMAGES_SCHEDULEDWORKFLOW=quay.io/opendatahub/ds-pipelines-scheduledworkflow@sha256:31d049e74ab038f3a6d3ff9fa8953a4d0ddb21b0efc43fbb5b07fbaf83817022
-IMAGES_MLMDENVOY=quay.io/opendatahub/ds-pipelines-metadata-envoy@sha256:f2d5d430bbc925520f635f35698e604aae391ace39b15a5d601a9c9eb26dec2b
-IMAGES_MLMDGRPC=quay.io/opendatahub/ds-pipelines-metadata-grpc@sha256:2490aadb2227cc72fd9e698549a8cd3270b669a2faa24bb0603c37f1c71ac8c4
-IMAGES_MLMDWRITER=quay.io/opendatahub/ds-pipelines-metadata-writer@sha256:89fc26374f8e58384628f6b178eb9b8e3ebb111fe395c529d0b65ba8adaa89f5
-IMAGES_DSPO=quay.io/opendatahub/data-science-pipelines-operator@sha256:c1d77b668149396a4409926eea279647c817a02868a3d21f9a4b5f30c1e86766
-IMAGES_CACHE=registry.access.redhat.com/ubi8/ubi-minimal@sha256:e52fc1de73dc2879516431ff1865e0fb61b1a32f57b6f914bdcddb13c62f84e6
-IMAGES_MOVERESULTSIMAGE=registry.access.redhat.com/ubi8/ubi-micro@sha256:443db9a646aaf9374f95d266ba0c8656a52d70d0ffcc386a782cea28fa32e55d
-IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103@sha256:6c3ae581b754017b335a70388c0010cf729df8a29daeb6651642ebee4e8abfde
-IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy@sha256:ab112105ac37352a2a4916a39d6736f5db6ab4c29bad4467de8d613e80e9bb33
+IMAGES_APISERVER=quay.io/opendatahub/ds-pipelines-api-server:latest
+IMAGES_ARTIFACT=quay.io/opendatahub/ds-pipelines-artifact-manager:latest
+IMAGES_PERSISTENTAGENT=quay.io/opendatahub/ds-pipelines-persistenceagent:latest
+IMAGES_SCHEDULEDWORKFLOW=quay.io/opendatahub/ds-pipelines-scheduledworkflow:latest
+IMAGES_MLMDENVOY=quay.io/opendatahub/ds-pipelines-metadata-envoy:latest
+IMAGES_MLMDGRPC=quay.io/opendatahub/ds-pipelines-metadata-grpc:latest
+IMAGES_MLMDWRITER=quay.io/opendatahub/ds-pipelines-metadata-writer:latest
+IMAGES_DSPO=quay.io/opendatahub/data-science-pipelines-operator:latest
+IMAGES_CACHE=registry.access.redhat.com/ubi8/ubi-minimal:8.8
+IMAGES_MOVERESULTSIMAGE=registry.access.redhat.com/ubi8/ubi-micro:8.8
+IMAGES_MARIADB=registry.redhat.io/rhel8/mariadb-103:1
+IMAGES_OAUTHPROXY=registry.redhat.io/openshift4/ose-oauth-proxy:v4.10
```

Comment:
for ose-oauth-proxy at least, the idea was to not only have a certain label of the image v4.10, but a set digest, pinned down to a point in time, so really this should always be in digest format. See https://github.com/opendatahub-io/odh-manifests/issues/760 Cause the digest could change with that kind of what I think is called a moving/floating tag.

Reply:
> convert these to digests via automated workflow (the digest that matches with that tag at the time of release)

good point, I do see the main point that a certain and common 4.x version tag is most important for the ODH project as a whole. Now, if all subprojects of ODH worked your way, i.e. determining what lands in odh-manifests in the end at a given time, then the point-in-time resolve of the most current digest to go along with v4.10 or whatever agreed-upon version of ose-oauth / openshift is actually not a bad thing. Up to now, I did not see that Github actions mechanism in i.e. model-mesh or dashboard, which is why the odh-manifests PR exists. Basically, I just want to ensure that the digest-values of ose-oauth-proxy in odh-manifests are the same across all components. How that is best achieved is up for discussion, your way seems very elegant.

@LaVLaS @andrewballantyne @rimolive @VaishnaviHire what do you think?

Reply:
I've updated it to include the sha for oauth proxy in `main` branch, as well as the added an option in the workflow to override any entries, when we automate this portion we'll fetch new digests for all tags, with the exception of oauth proxy which we will `--override` to keep it in sync with odh, if in the feature we can implement a similar workflow in other parts of odh then we can remove this flag, and update digests for oauth proxy with every release if needed.